app/views/my_modules/results.html.erb

```diff
@@ -48,9 +48,7 @@
 
   <div id="results" data-module-id="<%= @my_module.id %>">
   <% ordered_result_of(@my_module).each do |result| %>
-    <% cache [current_user, result] do %>
-      <%= render partial: "result", locals: { result: result } %>
-    <% end %>
+    <%= render partial: "result", locals: { result: result } %>
```

Comment:
Is this really needed?

Reply:
Smart annotations(it is just signature in the text), so when object, which is referenced by SA, is changed, it is needed to invalidate cache for step/result containing such SA. But in order to find those steps/results it is needed to scan them all, and it is unacceptable.